src/partials/head-meta.hbs

```diff
@@ -12,7 +12,7 @@
     <meta property="og:url" content="{{page.canonicalUrl}}">
     <meta property="og:title" content="{{page.title}} :: {{site.title}}">
     <meta property="og:description" content="{{page.description}}">
-    <meta property="og:image" content="https://docs.datastax.com/en/astra/_/img/datastax-docs-banner.png">
+    <meta property="og:image" content="{{{site.url}}}{{{site.ui.url}}}/img/datastax-docs-banner.png">
```

Comment:
I think double moustache would work here but triple is fine too in this case

Reply:
Maybe we go with double mustaches then? Although it's not really a security threat in this case, it's probably a good practice to only use triple when necessary?

Reply:
Maybe we go with double mustaches then? Although it's not really a security threat in this case, it's probably a good practice to only use triple when necessary?

Are triple mustaches indeed required for the rest of the variables in this PR?

```suggestion
    <meta property="og:image" content="{{site.url}}{{site.ui.url}}/img/datastax-docs-banner.png">
```